Formula/cbmc@5.56.0.rb

```diff
@@ -0,0 +1,52 @@
+class CbmcAT5560 < Formula
+  desc "C Bounded Model Checker"
+  homepage "https://www.cprover.org/cbmc/"
+  url "https://github.com/diffblue/cbmc.git",
+      tag:      "cbmc-5.56.0",
+      revision: "8a41865d97e180e3ca8e4049ee7978b07e428359"
+  license "BSD-4-Clause"
+
+  bottle do
```

Comment:
Why does this contain bottle information?

Reply:
Removed it. The difference w.r.t to the 5.55.0 formula is now just the version:
```
$ diff Formula/cbmc@5.55.0.rb Formula/cbmc@5.56.0.rb 
1c1
< class CbmcAT5550 < Formula
---
> class CbmcAT5560 < Formula
5,6c5,6
<       tag:      "cbmc-5.55.0",
<       revision: "598c3dac53c56ebdbb4d726d4093856301732930"
---
>       tag:      "cbmc-5.56.0",
>       revision: "8a41865d97e180e3ca8e4049ee7978b07e428359"
```